fv3core/decorators.py

```diff
@@ -350,6 +350,10 @@ def __init__(self, func, origin, domain, **kwargs):
         )
 
     def __call__(self, *args, **kwargs) -> None:
+        if "origin" in kwargs:
+            raise ValueError("cannot pass origin to FixedOriginStencil at call time")
+        if "domain" in kwargs:
+            raise ValueError("cannot pass domain to FixedOriginStencil at call time")
```

Comment:
Thanks for the heads up. I'll update it to this version when I merge, since it should be more informative (someone might not realize they're calling a FixedOrigin stencil).